src/components/unlocker.js

```diff
@@ -54,36 +55,31 @@ export function unlockPlayerResponse(playerResponse) {
 
 function getUnlockedPlayerResponse(videoId, reason) {
     // Check if response is cached
-    if (responseCache.videoId === videoId) return responseCache.playerResponse;
-
-    let playerResponse = useInnertubeEmbed();
-
-    if (playerResponse?.playabilityStatus?.status !== "OK") playerResponse = useProxy();
-
-    // Cache response for 10 seconds
-    responseCache = { videoId, playerResponse };
-    setTimeout(() => { responseCache = {} }, 10000);
-
-    return playerResponse;
+    // if (responseCache.videoId === videoId) return responseCache.playerResponse;
 
     // Strategy 1: Retrieve the video info by using a age-gate bypass for the innertube API
     // Source: https://github.com/yt-dlp/yt-dlp/issues/574#issuecomment-887171136
-    function useInnertubeEmbed() {
-        logger.info("Trying Unlock Method #1 (Innertube Embed)");
-        return innertubeClient.getPlayer(videoId)
-    }
+    logger.info("Trying Unlock Method #1 (Innertube Embed)");
+    let playerResponse = innertube.getPlayer(videoId);
 
     // Strategy 2: Retrieve the video info from an account proxy server.
     // See https://github.com/zerodytrash/Simple-YouTube-Age-Restriction-Bypass/tree/main/account-proxy
-    function useProxy() {
+    if (playerResponse?.playabilityStatus?.status !== "OK")
         logger.info("Trying Unlock Method #2 (Account Proxy)");
-        return proxy.getPlayerFromAccountProxy(videoId, reason);
-    }
+        playerResponse = proxy.getPlayer(videoId, reason);
+
+    // Cache response for 10 seconds
+    // responseCache = { videoId, playerResponse };
+    // setTimeout(() => { responseCache = {} }, 10000);
```

Comment:
The cache has no obvious effect, why is this here in the first place? 
On top of that, the cache is useless after this function gets called multiple times, since we never clear the timeout.

Reply:
there are often two unlocks. inital data and player request. to avoid two requests to youtube or my proxy, the cache was implemented.

Reply:
Can you point me to where those unlocks happen for both? Because from what i can see is that it can be called only once for the video. `getUnlockedPlayerResponse` only gets called once by `unlockPlayerResponse` which can only get called in `main.js` also once.

Reply:
Have tested it. Currently no second player request seems to be executed after the initial unlock. Only with the embedded player, there is another request after you click play.
https://kworb.net/youtube/video/Mi3GdCyzHCY.html

I think it would be good to keep the cache in case something unexpected happens (e.g. an endless loop during JSON interception).

Reply:
The embedded player does not work for me.

![image](https://user-images.githubusercontent.com/38593134/132969346-0bf40170-9e49-46b9-ac7f-f90e771966a2.png)

The cache does not work correctly at the moment and would not work at all for the endless loop case. So I have to fix it first.

Reply:
Can I just remove the setTimeout?

Reply:
yes, you can replace it with something else which fulfills the same purpose.

> The embedded player does not work for me.

strange. have you tried with the current version?

Reply:
> yes, you can replace it with something else which fulfills the same purpose.

Removed the `SetTimeout` because it had no use. the cache already is meant to be reassigned if the video does not match cache id.
> > The embedded player does not work for me.
> 
> strange. have you tried with the current version?

Current version works fine now :)